web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -34,19 +44,11 @@ interface urlList {
 
 export interface MediaValues {
   'regen:previewPhoto'?: urlType;
+  'regen:creditText'?: string;
   'regen:galleryPhotos'?: urlList;
-  'regen:landStewardPhoto'?: urlType;
   'regen:videoURL'?: urlType;
 }
 
-type valueObject = { '@value'?: string };
-export interface MediaValuesErrors {
-  'regen:previewPhoto'?: valueObject;
-  'regen:galleryPhotos'?: string;
-  'regen:landStewardPhoto'?: valueObject;
-  'regen:videoURL'?: valueObject;
-}
```

Comment:
let's maybe not get rid of this yet
I believe we should maintain the 2 form versions for now, just as I did for the Roles form